.github/workflows/deploy-dev.yaml

```diff
@@ -0,0 +1,65 @@
+name: deploy-dev
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+      - ".gitignore"
+      - "LICENCE"
+      - "CODEOWNERS"
+    branches:
+      - master
+env:
+  docker_image: docker.pkg.github.com/${{ github.repository }}/melosys-eessi:${{ github.sha }}
+jobs:
+  build:
+    name: Build, deploy to dev, and draft release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout latest code
+        uses: actions/checkout@v1
```

Comment:
Vi bør vel ta i bruk nyere versjon av checkout og setup-java actions. Enten @latest (men da kan bygg plutselig feile etter breaking change) eller siste major version (og da må vi oppdatere manuelt)

Reply:
> > > Gir: `fatal: No names found, cannot describe anything.` Vi trenger en første tag til å ta dette i bruk.
> > 
> > 
> > Hvordan får dere testet dette? thinking
> 
> Det var bare å kjøre `git describe --abbrev=0 --tags`. Var det faktisk det du spurte om?

Mente selve PRen, for å bekrefte at det fungerer før det merges